docs/toilAPI.rst

```diff
@@ -0,0 +1,72 @@
+Toil API
+********
+
+Job Methods
+-----------
+Jobs are the units of work in Toil which are composed into workflows.
+
+.. autoclass:: toil.job.Job
+   :members:
+
+Job.FileStore
+-------------
+The FileStore is an abstraction of a Toil run's shared storage.
+
+.. autoclass:: toil.job::Job.FileStore
+   :members:
+
+Job.Runner
+----------
+The Runner contains the methods needed to configure and start a Toil run.
+
+.. autoclass:: toil.job::Job.Runner
+   :members:
+   
+Job.Service
+-----------
+The Service class allows databases and servers to be spawned within a Toil workflow.
+
+.. autoclass:: toil.job::Job.Service
+   :members:
+   
+FunctionWrappingJob
+-------------------
+The subclass of Job for wrapping user functions.
```

Comment:
I'm a little unclear what you want to see here. There is documentation with the toil/job.py module for each of these that is embedded here. 

I've tended to not duplicate documentation, and tried to cover the use of the functions in the tutorial, so I'd prefer not to duplicate too much, but (obviously) I do want to make sure it's clear.